### PR TITLE
implement truncate for spaces

### LINKED ISSUE #91: [2pt] Truncate
Sometimes we need to truncate all data in all spaces. 

```lua
local ok, err = crud.truncate(space_name, opts)
```

where:

* `space_name` (`string`) - name of the space
* `opts`:
  * `timeout` (`?number`) - `vshard.call` timeout (in seconds)

### PR DESCRIPTION
**Pull request by dokshina**
This patch implements "crud.truncate" call. It's needed for spaces
truncation.

I didn't forget about

- [x] Tests
- [x] Changelog
- [x] Documentation

Closes #91

### CODE REVIEW

File: test/integration/truncate_test.lua [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                space:truncate()
+            end
+        ]])
+    end
+end
+
+g_memtx.before_each(function() before_each(g_memtx) end)
+g_vinyl.before_each(function() before_each(g_vinyl) end)
+
+local function add(name, fn)
```
Comment (dokshina) [inline test/integration/truncate_test.lua:79]: Sometimes this function should become a helper...

Comment (olegrok) [reply test/integration/truncate_test.lua:79]: I think we've discussed it before

**Comment by dokshina** [comment]
Please, update the doc and changelog.

**Review by olegrok — APPROVED** [review]
Thanks for your patch! LGTM.

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   selecting by equality of partially specified multipart primary index
   value was misinterpreted as a selecting by fully specified key value.
 
+### Added
+* `truncate` operation
```
Comment (dokshina) [inline CHANGELOG.md:18]: Please, add an empty line.

**Comment by dokshina** [comment]
Yes, it really looks good. Please, fix my comment and feel free to merge.

File: CHANGELOG.md (hunk 2 above)
Comment (olegrok) [reply CHANGELOG.md:19]: Added

**Pull request merged**